### PR TITLE
AddItemModal actually adds items

### PR DESCRIPTION
**Pull request by AaronSalo**
Items are added after done button is pressed. Still need to do error
checking (eg. no name filled) and reset the tag icons to unselected and
other similar stuff.

### CODE REVIEW

File: PantryPal/package-lock.json [hunk 1]
```diff
@@ -1,8 +1,9706 @@
 {
   "name": "pantrypal",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```
Comment (kyl-dc) [inline PantryPal/package-lock.json:6]: Should we update the git ignore to remove this file too? or is this meant to be included?

**Review by kyl-dc — COMMENTED** [review]
Looks great!
Just added a comment about adding package-lock to the git ignore

File: PantryPal/package-lock.json (hunk 1 above)
Comment (AaronSalo) [reply PantryPal/package-lock.json:6]: Yeah i think that would be beneficial

**Review by kyl-dc — APPROVED** [review]

File: PantryPal/package-lock.json (hunk 1 above)
Comment (AaronSalo) [reply PantryPal/package-lock.json:6]: NVM, looking into it, we just commit it every time, but ignore it
[source](https://dev.to/adamklein/package-lock-json-in-git-or-not-50l5)

**Pull request merged**